src/services/item/plugins/published/plugins/search/meilisearch.ts

```diff
@@ -89,15 +91,15 @@ export class MeiliSearchWrapper {
   }
 
   private async logTaskCompletion(task: EnqueuedTask, itemName: string) {
-    const finishedTask = await (await this.getIndex()).waitForTask(task.taskUid);
-    if (task.status === TaskStatus.TASK_SUCCEEDED) {
-      this.logger.info(
-        `Meilisearch task completed: item name => ${itemName}, task type => ${task.type}, result: Successful task`,
-      );
-    } else {
+    try {
+      const finishedTask = await (await this.getIndex()).waitForTask(task.taskUid);
       this.logger.info(
-        `Meilisearch task completed: item name => ${itemName}, task type => ${task.type}, result: ${finishedTask.error}`,
+        `Meilisearch ${task.type} task completed (${task.status}): item name => ${itemName}. ${
+          finishedTask.error ? `Error: ${finishedTask.error}` : ''
+        }`,
       );
+    } catch (e) {
+      // catch timeout while waiting
```

Comment:
Should we print something here?

Reply:
I'm not sure, I can add something, I just don't want to make it confusing because a timeout does not mean the task will fail